client/src/app/shared/forms/form-validators/video-validators.service.ts

```diff
@@ -92,5 +93,10 @@ export class VideoValidatorsService {
         'required': this.i18n('A date is required to schedule video update.')
       }
     }
+
+    this.VIDEO_PRODUCED_AT = {
+      VALIDATORS: [ ],
```

Comment:
Nothing put here at the moment but we may add something.

Reply:
when downloading/importing a video it uses youtube-dl to do it from many different providers. From youtube the video info json have these date field `"upload_date": "20100503"`, some other providers like bitchute don't give an `upload_date` or they may give it in a different format other than `Ymd`, so really I'm not sure we should put any validators, we should just try to parse it in the many different formats it can come in, if it exists and it not then oh well

Reply:
maybe on top of having the `producedAt` field we could also have an `uploadedAt` field in that case.

Reply:
Thanks for the review. This is more about specification so I suggest we go to the issue's discussion : https://github.com/Chocobozzz/PeerTube/issues/946

Reply:
> when downloading/importing a video it uses youtube-dl to do it from many different providers. From youtube the video info json have these date field `"upload_date": "20100503"`, some other providers like bitchute don't give an `upload_date` or they may give it in a different format other than `Ymd`, so really I'm not sure we should put any validators, we should just try to parse it in the many different formats it can come in, if it exists and it not then oh well

We may do it if it is decided to save the date when a video was uploaded on another platform.

Reply:
If the date is available from the other platform, then lets use it, if not then just ignore it. I'm thinking it will be useful for users to know when the video was uploaded so they can know if they've already watched it before, in case the same video is on multiple instances they follow they can use that as a metric to determine if it's a new upload or not.